icu4c/source/common/locid.cpp

```diff
@@ -711,6 +713,178 @@ Locale::setDefault( const   Locale&     newLocale,
     locale_set_default_internal(localeID, status);
 }
 
+Locale U_EXPORT2
+Locale::forLanguageTag(StringPiece tag, UErrorCode& status)
```

Comment:
Optional: I prefer the name "errorCode" for the variable of type "UErrorCode". But I know "status" is also common.